requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy >= 1.16.6
+protobuf >= 3.16.0
```

Comment:
Based on discussion with @askhade @jcwchen to remove the minimum version requirement here for consistency with past releases. Also check protobuf version requirements of tf/torch.

Reply:
Thanks for pointing out! In some converters (onnx-tensorrt, onnx-mlir) they do have protobuf version requirements... To grant users flexibility for using different versions of protobuf, remove the minimum version requirement for ONNX here.